### PR TITLE
Reorganizing the scenario tests as described in Issue #50.

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
Please see Issue #50 for additional details.

### CODE REVIEW

**Comment by hongdai** [comment]
I pulled the changes and was able to run the tests successfully on my machine!

However, it is much slower than before because we have many more projects to build and run. How about we create only one project at the scenario root level and include all tests from the subdirectories?

On the console, it will only output total tests passed/failed info, individual test result is available in the testsResults.xml file.

File: src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "xunit": "2.0.0-beta5-build2785",
+    "xunit.abstractions.netcore": "1.0.0-prerelease",
+    "xunit.assert": "2.0.0-beta5-build2785",
+    "xunit.core.netcore": "1.0.1-prerelease",
+    "xunit.netcore.extensions":  "1.0.0-prerelease-*"
+  },
+  "frameworks": {
+    "dnxcore50": {}
+  }
+}
```
Comment (roncain) [inline src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/project.json]: Nit: GitHub will show this icon whenever the file is missing a final newline, so ultimately we need a newline here (and anywhere else this icon shows).

**Comment by roncain** [comment]
LGTM

I recommend we merge as soon as we get another confirmation.

I also verified I can build and run this PR, and the CI machine's OuterLoop run succeeded.  I also verified the SLN solution folder layout looks good and that you can build everything from VS.

Nit: GitHub wants a newline at the end of every file, and some of these are missing them.  But I see this was already the case in some of the renamed and copied files.  I recommend we deal with that as a separate Issue+PR so we get all of them at once, including any others in existing code.

The normal way to close the original issue would be to include "Fixes #nnn" as the last line in the PR commit message.  No need to change this PR, because we can manually close the issue and link back to this PR.

I recommend we do *not* make a common csproj to shorten build time.  That is the job of the build tool work being discussed at https://github.com/dotnet/corefx/issues/1477.  I prefer we organize for clarity, like this PR does.

Next step is to ensure we have open issues for file and method renaming and any other scenario cleanup work

**Comment by mconnew** [comment]
LGTM :shipit:

**Pull request merged**